src/routes/web/searchResults.ts

```diff
@@ -27,6 +27,11 @@ const searchResultsRoutes = [
     path: webRoutePaths.getResults,
     handler: SearchResultsController.getSearchResultsHandler,
   },
+  {
+    method: 'POST',
```

Comment:
As discussed, we need to deep link the search experience. so please start using the GET method.

Reply:
On the results page, summary items, filters, and map results are being rendered from ajax now since all the search criteria are stored locally. Now we need to eliminate the local storage and display all search criteria in the URL as query parameters. Starting with the first step of the guided search, this should be done. This change requires a lot of coding changes. Do you want me to include that in this ticket?

Reply:
Yes, obviously as part of activities performing from this sprint we need to focus on deep linking.

Reply:
Now we have URL query params